app/src/main/res/values/strings.xml

```diff
@@ -179,6 +179,7 @@
     <string name="url_message_details_offline_backends_learn_more" translatable="false">https://support.wire.com/hc/articles/9357718008093-Backend</string>
     <string name="url_self_client_verification_learn_more" translatable="false">https://support.wire.com/hc/en-us/articles/207693005-How-can-I-verify-my-own-devices-</string>
     <string name="url_self_client_fingerprint_learn_more" translatable="false">https://support.wire.com/hc/en-us/articles/207692235-How-can-I-compare-key-fingerprints-</string>
+    <string name="url_federation_support" translatable="false">https://support.wire.com/hc/en-us/categories/4719917054365-Federation</string>
```

Comment:
```suggestion
    <string name="url_federation_support" translatable="false">https://support.wire.com/hc/categories/4719917054365-Federation</string>
```

I think we need to use the `un-localized `version of the link here, we have several actually that we need to remove `/en-us` 😅  (not related to federation)